validator/src/main.rs

```diff
@@ -370,6 +370,22 @@ fn hardforks_of(matches: &ArgMatches<'_>, name: &str) -> Option<Vec<Slot>> {
     }
 }
 
+fn get_observable_vote_accounts(matches: &ArgMatches<'_>) -> HashSet<Pubkey> {
```

Comment:
(bikeshed, but I think this is a bit clearer)
```suggestion
fn get_vote_accounts_to_monitor(matches: &ArgMatches<'_>) -> HashSet<Pubkey> {
```